### PR TITLE
fix: Adds ignoring .md in sub-dirs for integration build

### LINKED ISSUE #1032: Add ignoring MD files in integration build for sub-dirs
Hey folks, 

Please, add ignoring .MD files in integration build for all levels of directories.

### PR DESCRIPTION
**Pull request by amandakarina**
Hey folks, 
This pull request fixes #1032

### CODE REVIEW

File: infra/terraform/test-org/ci-triggers/triggers.tf [hunk 1]
```diff
@@ -65,10 +65,11 @@ resource "google_cloudbuild_trigger" "tf_validator" {
     }
   }
   substitutions = {
-    _TEST_PROJECT = local.tf_validator_project_id
+    _TERRAFORM_VERSION = "0.12.31"
+    _TEST_PROJECT      = local.tf_validator_project_id
   }
 
-  filename = "build/int.cloudbuild.yaml"
+  filename = ".ci/cloudbuild-tests-integration.yaml"
```
Comment (bharathkkb) [inline infra/terraform/test-org/ci-triggers/triggers.tf:72]: @melinath FYI the tfv trigger seemed to have changed from what have here. I have updated to current values so there is no diff.

```
  # google_cloudbuild_trigger.tf_validator will be updated in-place
  ~ resource "google_cloudbuild_trigger" "tf_validator" {
      ~ description    = "Pull request build for tf-validator with terrafom 12" -> "Pull request build for tf-validator"
      ~ filename       = ".ci/cloudbuild-tests-integration.yaml" -> "build/int.cloudbuild.yaml"
        id             = "cloud-foundation-cicd/8d4c3be2-77a2-47ab-b95b-05749c15b2cd"
        name           = "tf-validator-tests-tf-12"
      ~ substitutions  = {
          - "_TERRAFORM_VERSION" = "0.12.31" -> null
            # (1 unchanged element hidden)
        }
        # (6 unchanged attributes hidden)


        # (2 unchanged blocks hidden)
    }
```

**Review by bharathkkb — APPROVED** [review]

**Comment by bharathkkb** [comment]
This has been applied

**Comment by bharathkkb** [comment]
Looks like its WAI, thanks for catching this @amandakarina

File: infra/terraform/test-org/ci-triggers/triggers.tf (hunk 1 above)
Comment (melinath) [reply infra/terraform/test-org/ci-triggers/triggers.tf:72]: Thanks, I didn't realize that this was managed via terraform!

**Pull request merged**